data/transport missions.txt

```diff
@@ -3469,3 +3469,114 @@ mission "Inheritance Redirection"
 				decline
 	on complete
 		dialog "There is no inheritance for Mr. Pappa waiting for you at the bank, but there are many other victims that paid some credits while expecting to earn a larger amount of credits in return. Some make a big fuss about it, until the alerted authorities start fining them for attempted inheritance fraud. You leave the bank in silence."
+
+
+
+mission "Hauler VI cargo prototype 1"
+	name "Escort Hauler prototype to formal dinner"
+	description "Escort the Hauler prototype to <destination> to deliver sushi to a formal dinner."
```

Comment:
`to` used three times in rapid succession. `for a formal dinner` maybe?

Reply:
Will use `for`.